nautobot/extras/models/tags.py

```diff
@@ -83,6 +83,8 @@ def validate_content_types_removal(self, content_types_id):
 
 
 class TaggedItem(BaseModel, GenericUUIDTaggedItemBase):
+    # FixME: changing related name to tagged_items causes a
+    #  "ValueError: The annotation 'tagged_items' conflicts with a field on the model."
```

Comment:
This is okay because this is actually used as an intermediate model when associating `Tag` to a model, and the `related_name` is already being set in the `TaggableManager`. We can skip this one in other words.